app/tests/edit_clear_all.test.js

```diff
@@ -0,0 +1,53 @@
+import add from '../modules/add.js';
+import checkState from '../modules/checkboxes.js';
+import clearCompleted from '../modules/clear_all_completed.js';
+import editTask from '../modules/edit_task.js';
+
+describe('Tests for editing, clearing all completed tasks and checking tasks', () => {
```

Comment:
# Congratulations
![Capture](https://user-images.githubusercontent.com/63972670/194705060-104dcf1b-a658-4d18-ab87-ef3d50324098.JPG)

Reply:
#### Hi @OLIPLICHE, Thanks for the feedback.🙃